mk/cfg/x86_64-unknown-linux-uclibc.mk

```diff
@@ -0,0 +1,29 @@
+# x86_64-unknown-linux-uclibc configuration
```

Comment:
So, is your decision to preference using rustbuild only? Because I need support for using rust with buildroot, and I think the way the buildroot feature/rust-v5 branch has it is bootstrapping from the bottom up.